tests/test_simple_harvest.py

```diff
@@ -77,8 +78,17 @@ def test_simple_harvest(
     chain.sleep(60 * 60 * hours)
     chain.mine(1)
 
+    # Check that there's CRV in the gauge
+    gaugeCrvAmount = rewardToken.balanceOf(gauge)
+
+    if gaugeCrvAmount == 0:
+        warn("No CRV in the gauge that can be claimed by gauge token holders.")
+    else:
+        # If there's CRV in the gauge, then we must have a non-zero claim
+        stratClaimableCrv = gauge.claimable_reward.call(strategy, rewardToken)
```

Comment:
if there's CRV available in the gauge, the strategy must have a non-zero value to claim